website/source/docs/guides/performance.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "docs"
+page_title: "Server Performance"
+sidebar_current: "docs-guides-performance"
+description: |-
+  Consul requires different amounts of compute resources, depending on cluster size and expected workload. This guide provides guidance on choosing compute resources.
+---
+
+# Server Performance
+
+Since Consul servers run a [consensus protocol](/docs/internals/consensus.html) to
+process all write operations and are contacted on nearly all read operations, server
+performance is critical for overall throughput and health of a Consul cluster. Servers
+are generally I/O bound for writes because the underlying Raft log store performs a sync
+to disk every time an entry is appended. Servers are generally CPU bound for reads since
+reads work from a fully in-memory data store that is optimized for concurrent access.
+
+<a name="minimum"></a>
+## Minimum Server Requirements
+
+In Consul 0.7, the default server [performance parameters](/docs/agent/options.html#performance)
+were tuned to allow Consul to run reliably (but relatively slowly) on a server cluster of three
+[AWS t2.micro](https://aws.amazon.com/ec2/instance-types/) instances. These thresholds
+were determined empirically using a leader instance that was under sufficient read, write,
+and network load to cause it to permanently be at zero CPU credits, forcing it to the baseline
+performance mode for that instance type. Real-world workloads typically have more bursts of
+activity, so this is a conservative and pessimistic tuning strategy.
+
+This default was chosen based on feedback from users, many of whom wanted a low cost way
+to run small production or development clusters with low cost compute resources, at the
+expense of some performance in leader failure detection and leader election times.
+
+The default performance configuration is equivalent to this:
+
+```javascript
+{
+  "performance": {
+    "raft_multiplier": 5
+  }
+}
+```
+
+<a name="production"></a>
+## Production Server Requirements
+
+When running Consul 0.7 and later in production, it is recommended to configure the server
```

Comment:
Do we necessarily recommend this? It seems like the default is good as it will eliminate unnecessary elections, while trading some latency for true leader failure cases.